pkgs/development/python-modules/deltalake/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyarrow
+, pyarrow-hotfix
+, rustPlatform
+, perl
+, pytestCheckHook
+, pytest-benchmark
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "deltalake";
+  version = "0.18.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "delta-io";
+    repo = "delta-rs";
+    rev = "refs/tags/python-v${version}";
+    hash = "sha256-R+fyxNbPhS04G7jWPopFNIWmZ4VW0SV0JKa4zBcV3Vg=";
+  };
+
+  postPatch = ''
+    # No Cargo.lock file in source
+    ln -s ${./Cargo.lock} Cargo.lock
+ '';
+
+ cargoDeps = rustPlatform.importCargoLock {
+   lockFile = ./Cargo.lock;
```

Comment:
Could we just use `rustPlatform.fetchCargoTarball` (Since we don't need git hash)?

Reply:
@Aleksanaa thank you for the suggestion. I have have got it to work with `rustPlatform.fetchCargoTarball` and `fetchPypi`. Thanks become mutch better :-)

Have added changes as commits, and will squash when you are had a look.